File: app/src/main/java/com/wealoha/social/beans/NotifyGetData.java
package com.wealoha.social.beans;

import java.util.List;
import java.util.Map;

/**
 * 
 * 
 * @author javamonk
 * @createTime 2015年2月25日 上午10:33:05
 */
public class NotifyGetData extends ResultData {

	public Map<String, Integer> commentCountMap;
	public Map<String, Integer> likeCountMap;
	public Map<String, UserDTO> userMap;
	public Map<String, ImageCommonDto> imageMap;
	public Map<String, PostDTO> postMap;
	public List<AbsNotify2DTO> list;
	public String nextCursorId;
}

File: app/src/main/java/com/wealoha/social/beans/NotifyGetData.kt
package com.wealoha.social.beans

/**
 *
 *
 * @author javamonk
 * @createTime 2015年2月25日 上午10:33:05
 */
data class NotifyGetData(
    var commentCountMap: Map<String, Int>? = null,
    var likeCountMap: Map<String, Int>? = null,
    var userMap: Map<String, UserDTO>? = null,
    var imageMap: Map<String, ImageCommonDto>? = null,
    var postMap: Map<String, PostDTO>? = null,
    var list: List<AbsNotify2DTO>? = null,
    var nextCursorId: String? = null
) : ResultData() {

}